src/tests/multihost/alltests/test_kcm.py

```diff
@@ -23,11 +27,11 @@ class TestKcm(object):
     @pytest.mark.tier1_2
     def test_client_timeout(self, multihost, backupsssdconf):
         """
-        :Title: kcm: Increase client idle
-        timeout to 5 minutes
-
-        @bugzilla:
-        https://bugzilla.redhat.com/show_bug.cgi?id=1884205
+        :title: kcm: Increase client idle
+         timeout to 5 minutes
+        :id: 6933cb85-1616-4b7f-a049-e81ab4c05347
+        :bugzilla:
+         https://bugzilla.redhat.com/show_bug.cgi?id=1884205
```

Comment:
Entire test script does not have any :steps: and :expectedresults:

Reply:
Change is only to update docstrings for auto import